__tests__/app.test.js

```diff
@@ -40,4 +40,23 @@ describe('testing for get /api/topics', () => {
             expect(result.body.message).toBe('Path not found')
         })
     })
+})
+
+describe('tests for get /api', () => {
+    test('returns status code 200', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+    })
+    test('returns an object of available endpoints and their descriptions', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then((result) => {
+            const {endpoints} = result.body
+            expect(typeof endpoints).toBe('object')
+            //expect(Object.keys(endpoints).length).toBe(3)
+            expect(endpoints['GET /api/topics'].description).toBe('serves an array of all topics')               
```

Comment:
To make the test more robust you could check that all the properties exist on that object.

However, since we are going to be adding to this endpoint file throughout the week you could write one test that asserts that the response from your /api endpoint looks exactly like the endpoints file within your repo.